test/ngAnimate/animateSpec.js

```diff
@@ -459,6 +459,26 @@ describe('animations', function() {
       expect(commentNode[0].parentNode).not.toBe(parentNode);
     }));
 
+    it('enter() should animate a transcluded clone with `templateUrl`', function() {
+      module(function($compileProvider) {
+        $compileProvider.directive('foo', valueFn({templateUrl: 'foo.html'}));
+      });
+
+      inject(function($animate, $compile, $rootScope, $templateCache) {
+        parent.append(jqLite('<foo ng-if="showFoo"></foo>'));
+        $templateCache.put('foo.html', '<div>FOO</div>');
+        $compile(parent)($rootScope);
+
+        expect(capturedAnimation).toBe(null);
```

Comment:
Everywhere in other places it's only checked if `capturedAnimation` is falsy. We should be consistent.